js/style/style.js

```diff
@@ -247,19 +251,95 @@ Style.prototype = util.inherit(Evented, {
         zh.lastZoom = z;
     },
 
+    _checkLoaded: function () {
+        if (!this._loaded) {
+            throw new Error('Style is not done loading');
+        }
+    },
+
     /**
-     * Apply multiple style mutations in a batch
-     * @param {function} work Function which accepts the StyleBatch interface
+     * Apply queued style mutations in a batch
      * @private
      */
-    batch: function(work) {
-        styleBatch(this, work);
+    update: function(classes, options) {
```

Comment:
The terminology here is starting to feel overwhelming: `mutate`, `update`, `change`, `cascade`, `recalculate`...

How do you feel about picking a consistent verb (i.e. `update`, `change`) and differentiating these properties and methods with direct objects?

 - `cascade` -> `updateTransitions`
 - `recalculate` -> `updateZoom`
 - `mutations` -> `updates`
 - `broadcastLayers` -> `updateWorkerLayers`
 - `reloadSource` -> `updateSource`
 - `change` event -> `update`